packages/plugin-core/src/_extension.ts

```diff
@@ -665,7 +667,9 @@ export async function _activate(
         codeWorkspacePresent,
       });
 
-      warnConflictingExtensions();
+      if (extensionInstallStatus === InstallStatus.INITIAL_INSTALL) {
+        await warnIncompatibleExtensions({ ext: ws });
```

Comment:
nit: do we actually need to await here?  The rest of initialization doesn't need to be blocked on this warning right?

Reply:
right. it will be immediately resolved after the warning message is displayed, but worth to be correct.

Reply:
fixed